backend/api/Startup.cs

```diff
@@ -173,20 +173,31 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseAuthentication();
             app.UseAuthorization();
-            app.UseHealthChecks("/api/live", new HealthCheckOptions
+            app.UseHealthChecks("/api/live", 8080, new HealthCheckOptions
             {
                 Predicate = r => r.Name.Contains("liveliness"),
                 ResponseWriter = UIResponseWriter.WriteHealthCheckUIResponse
             });
-            app.UseHealthChecks("/api/ready", new HealthCheckOptions
+            app.UseHealthChecks("/api/ready", 8080, new HealthCheckOptions
```

Comment:
I'd love to see the PORT value come from configuration so we don't tie ourselves to 8080 if things change on Openshift or we decide to use another NET Core image